packages/webdriver-utils/src/providers/genericProvider.js

```diff
@@ -63,6 +69,18 @@ export default class GenericProvider {
     }
   }
 
+  async getInitialPosition() {
+    if (this.currentTag.osName === 'iOS') {
```

Comment:
can do this to lower case and check for `ios`. So that it doesn't get missed on some future upgrade that changes this osName.

Reply:
@prklm10  `iOS` is being set by us manually in the normalization layer.
Here - https://github.com/percy/cli/blob/b78c8d644e43da7eefce79b7a6ff8a9714d4398f/packages/webdriver-utils/src/metadata/normalizeData.js#L8

Reply:
got it 🍍.